Source/Core/Common/SFMLHelper.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <SFML/Network/Packet.hpp>
```

Comment:
Yes, because it doesn't drag the dependency directly in the include, preventing it from leaking a dependency on SFML if this header is ever included in another header in the future. It also potentially reduces the number of files that would need to be recompiled by way of transitive includes if SFML is ever updated and the packet header changes (since it's not directly including it). It doesn't matter how many things use [x].

Avoiding direct dependencies in headers on external library headers is always a good thing.

Reply:
Ah yeah, good point.